tests/clients/github/test_githubclient.py

```diff
@@ -24,9 +27,147 @@ def setUp(self):
+        repo = self.client.create_repo(name=self.repo_name, auto_init=True)
+        repo = self.client.get_repo(repo_full_name=repo.full_name)
+
+        self.info("set file to repo")
+        repo.set_file(path=f"{dir_name}/{f_name}.txt", content="test set file to repo")
+
+        self.info("download dir")
+        repo.download_directory(src="", download_dir="")
+        self.assertEqual(j.sals.fs.is_file(f"./tfttesting/{repo.name}/{dir_name}/{f_name}.txt"), True)
+        j.sals.fs.rmtree("./tfttesting")
+
+    def test05_github_create_milestoes(self):
+        """Test case for create milestones.
+
+        **Test Scenario**
+        #. create repo with auto init.
+        #. create milestones
+        #. check if milestones is created
+        #. delete milestones
+        #. check if milestones id deleted
+        """
+        self.info("create repo with auto init.")
+        self.repo_name = self.random_string()
+        title = self.random_string()
+        repo = self.client.create_repo(name=self.repo_name, auto_init=True)
+
+        self.info("create milestones")
+        ceated_milestone = repo.create_milestone(title=title)
+
+        self.info("check if milestones is created")
+        milestone = repo.get_milestone(number=ceated_milestone.number)
+        self.assertEqual(milestone.title, ceated_milestone.title)
+
+    def test06_github_create_issue(self):
+        """Test case for create issue.
+
+        **Test Scenario**
+        #. create repo with auto init.
+        #. create issue
+        #. check if issue is created
+        """
+
+        self.info("create repo with auto init")
+        self.repo_name = self.random_string()
+        issue_title = self.random_string()
+        repo = self.client.create_repo(name=self.repo_name, auto_init=True)
+        self.info("create issue")
+        issue_created = repo.create_issue(title=issue_title)
+
+        self.info("check if issue is created")
+        self.assertEqual(issue_created.title, repo.get_issue(issue_created.number).title)
+
+    def test06_github_issue_with_milestone(self):
+        """Test case for create issue with milestone.
+
+        **Test Scenario**
+        #. create repo with auto init.
+        #. create milestone
+        #. create issue with milestone
+        #. check if issue has milestone
+        """
+
+        self.info("create repo with auto init")
+        self.repo_name = self.random_string()
+        issue_title = self.random_string()
+        milestone_title = self.random_string()
+        repo = self.client.create_repo(name=self.repo_name, auto_init=True)
+
+        self.info("create milestone")
+        milestone = repo.create_milestone(title=milestone_title)
+
+        self.info("create issue with milestone")
+        issue_created = repo.create_issue(title=issue_title, milestone=milestone)
+
+        self.info("check if issue has milestone")
+        self.assertEqual(milestone.title, issue_created.milestone.title)
```

Comment:
Please check `milestone_title` against `issue_created.milestone.title`.

Reply:
here i need to check if issue (issue_created) have a milestone